apps/condo/domains/common/components/HeaderActions.tsx

```diff
@@ -1,9 +1,11 @@
 import { Tabs } from 'antd'
-import Router from 'next/router'
+import { useRouter } from 'next/router'
 import React from 'react'
 
 import { useIntl } from '@open-condo/next/intl'
 
+import { isSafeUrl } from '@condo/domains/common/utils/url.utils'
```

Comment:
Looks like you should set up autoformatting :)

Reply:
How I can do this?